packages/dashboard/src/components/tests/__snapshots__/tooltip.test.tsx.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
If we use enable on the `react-components` package we should delete this because AFAIK we are not  using snapshots con the `react-components` package